### PR TITLE
OSDOCS 12073: Updated release note to add user-defined labels and tags for GCP in GA

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.17

Issue: https://issues.redhat.com/browse/OSDOCS-12073

Link to docs preview: https://82437--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-installation-gcp-user-defined-label-tags_release-notes and https://82437--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-technology-preview-tables_release-notes

SME review:
- [x] SME has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -63,6 +63,14 @@ With this release, you can now use DNF to install additional packages to your cu
 [id="ocp-4-17-installation-and-update_{context}"]
 === Installation and update
 
+[id="ocp-4-17-installation-gcp-user-defined-label-tags_{context}"]
+==== User-defined labels and tags for GCP
+
+With this update, the user-defined labels and tags for {gcp-first} is Generally Available.
+
+For more information, see xref:../installing/installing_gcp/installing-gcp-customizations.html#installing-gcp-user-defined-labels-and-tags_installing-gcp-customizations[Managing user-defined labels and tags for GCP].
```
Comment (dfitzmau) [inline release_notes/ocp-4-17-release-notes.adoc:71]:
```suggestion
For more information, see xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-user-defined-labels-and-tags_installing-gcp-customizations[Managing user-defined labels and tags for GCP].
```

**Comment by xenolinux** [comment]
@subhtk Before I merge, just wanted to check if the PR awaits QE approval?

**Comment by subhtk** [comment]
cc @jianli-wei Can you ptal?

**Comment by xenolinux** [comment]
Merging now that we have QE ack!

**Pull request merged**